src/tests/ftest/cart/corpc/cart_corpc_five_node.py

```diff
@@ -37,7 +37,7 @@ class CartCoRpcFiveNodeTest(Test):
     """
     Runs basic CaRT CoRPC tests
 
-    :avocado: tags=all,corpc,five_node
+    :avocado: tags=all,all_cart,pr,corpc,five_node
```

Comment:
Why `all_cart` instead of just `cart`?

Reply:
Renamed `all_cart` to `cart`: b8ba1a4dd769bfb615c521f8d533049cc3ae49e4